src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -149,4 +160,62 @@ export class StudentCourseDetailsPageComponent implements OnInit {
   setDefaultPic(teammateProfile: StudentProfileWithPicture): void {
     teammateProfile.photoUrl = '/assets/images/profile_picture_default.png';
   }
+
+  /**
+   * Checks the option selected to sort teammates.
+   * @param by: option for sorting
```

Comment:
Could consider renaming `by` with `sortOption` for clarity, along with the rest of its uses in this file

Reply:
Of course, thank you for the comment!